### PR TITLE
Foo 005 rw1

### PR DESCRIPTION
**Pull request by Edman25**
## Checklist for Reviewing a Pre-Processing Script

- [ ] Does the python script contain the following 4 sections: Download data and save to your data directory, Process data, Upload processed data to Carto/Upload processed data to Google Earth Engine, Upload original data and processed data to Amazon S3 storage?
- [ ] Does the script have the standardized variable names for: dataset_name, raw_data_file, processed_data_file?
- [ ] Does the script create and use a 'data' directory?
- [ ] Does the script use a python module to automatically download the data? If this is not possible, are there explicit instructions for how to download the data (step by step with instructions about every input parameter and button to click to find the exact data) and does it use shutil to move the data from 'Downloads' into the data directory?
- [ ] Is the script automated as much as possible to minimize rewriting code the next time the dataset updates (ex 1: can you automatically pull out column names instead of typing them yourself? ex 2: if you are dropping columns with no data, did you use pandas to find the nodata columns instead of dropping the column by name?)
- [ ] Are there comments on almost every line of code to explicitly state what is being done and why?
- [ ] Are you uploading to the correct AWS location?
- [ ] For GEE scripts, did you explicitly define the band manifest with a pyramiding policy so that we can easily change it later if we need to?
- [ ] Does the README contain all relevant links?
- [ ] Does the README state the original file type?
- [ ] Does the README list all of the processing steps that were taken in the script?
- [ ] For netcdfs, does the README state which variables were pulled from the netcdf?
- [ ] Did you use the util functions whenever possible?
- [ ] Have you checked the processed data file on your computer to make sure it matches the data you uploaded to Carto? (spaces or symbol names in column titles often get changed in the upload process-please change these before uploading so that the backed up processed data matches the data on Carto)
- [ ] Does the folder name and python script file name match the dataset_name variable in the script? Are they all lowercase? Does the processing script end in '_processing.py'? (this part is often forgotten)

### CODE REVIEW

File: foo_005_rw1_crop_area_production/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Crop Land Area and Production Dataset Pre-processing
+
+This file describes the data pre-processing that was done to [the Crop Land Area and Production](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/PRFF8V&version=4.0) for [display on Resource Watch](in progress).
```
Comment (amsnyder) [inline foo_005_rw1_crop_area_production/README.md:3]: link to rw?

File: foo_005_rw1_crop_area_production/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+The data was provided by the source as two zip files containing multiple TIFF files. The files show the harvest and production areas for several crops and specific technologies. Only files encompassing maize,soybean, wheat and rice crops as well as all technologies were processed.
+
+Please see the [Python script](https://github.com/resource-watch/data-pre-processing/blob/master/foo_005_rw1_crop_area_production/foo_005_rw1_crop_area_production_processing.py) for more details on this processing.
+
+You can view the processed Crop Land Area and Production dataset [on Resource Watch](in progress).
```
Comment (amsnyder) [inline foo_005_rw1_crop_area_production/README.md:9]: link to rw?

File: foo_005_rw1_crop_area_production/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+You can also download the original dataset [from the source website](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/PRFF8V&version=4.0).
+
+###### Note: This dataset processing was done by [Eduardo Castillero Reyes](https://wrimexico.org/profile/eduardo-castillero-reyes), and QC'd by ....
```
Comment (amsnyder) [inline foo_005_rw1_crop_area_production/README.md:13]:
```suggestion
###### Note: This dataset processing was done by [Eduardo Castillero Reyes](https://wrimexico.org/profile/eduardo-castillero-reyes), and QC'd by [Amelia Snyder](https://www.wri.org/profile/amelia-snyder).
```

File: foo_005_rw1_crop_area_production/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+## Crop Land Area and Production Dataset Pre-processing
+
+This file describes the data pre-processing that was done to [the Crop Land Area and Production](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/PRFF8V&version=4.0) for [display on Resource Watch](in progress).
+
+The data was provided by the source as two zip files containing multiple TIFF files. The files show the harvest and production areas for several crops and specific technologies. Only files encompassing maize,soybean, wheat and rice crops as well as all technologies were processed.
```
Comment (amsnyder) [inline foo_005_rw1_crop_area_production/README.md:5]:
```suggestion
The data was provided by the source as two zip files containing multiple TIFF files. The files show the harvest and production areas for several crops, disaggregated by specific technologies. Only files encompassing maize, soybean, wheat, and rice crops were processed at the complete crop level were used (combining all technologies for a total).
```

**Review by amsnyder — CHANGES_REQUESTED** [review]

File: foo_005_rw1_crop_area_production/README.md (hunk 1 above)
Comment (Edman25) [reply foo_005_rw1_crop_area_production/README.md:3]: adding it

File: foo_005_rw1_crop_area_production/README.md (hunk 2 above)
Comment (Edman25) [reply foo_005_rw1_crop_area_production/README.md:9]: adding it

**Pull request merged**